pros/serial/devices/vex/v5_device.py

```diff
@@ -76,7 +76,8 @@ def natural_key(chunk: str):
             return [ports[0], *joystick_ports]
         else:
             raise ValueError(f'Invalid port type specified: {p_type}')
-    if len(joystick_ports) > 0 and p_type.lower() == 'system':
+    # these can now also be used as user ports
+    if len(joystick_ports) > 0:  # and p_type.lower() == 'system':
```

Comment:
Why is this commented out? If it's not needed can you remove it?

Reply:
i'd prefer we keep this in because this whole function is sorta brittle